dali/pipeline/pipeline.cc

```diff
@@ -354,22 +384,21 @@ int Pipeline::AddOperator(const OpSpec &const_spec, const std::string& inst_name
   for (int i = 0; i < spec.NumOutput(); ++i) {
     string output_name = spec.OutputName(i);
     string output_device = spec.OutputDevice(i);
-    string error_str = "(op: '" + spec.SchemaName() + "', output: '" +
-      output_name + "')";
 
     auto it = edge_names_.find(output_name);
-    DALI_ENFORCE(it == edge_names_.end(), "Output name '" +
-        output_name + "' conflicts with existing intermediate "
-        "result name. " + error_str);
+    DALI_ENFORCE(it == edge_names_.end(),
+                 make_string("Error for ", FormatOutput(spec, i), ". Output name \"", output_name,
+                             "\" conflicts with existing intermediate result name."));
```

Comment:
```suggestion
                 make_string("Error for ", FormatOutput(spec, i), ". Output name \"", output_name,
                             "\" conflicts with an existing intermediate result name."));
```
I'm not sure about the preposition near "Error", too - "Error for output 0" sounds awkward... Perhaps just "in"?
```suggestion
                 make_string("Error in ", FormatOutput(spec, i), ". Output name \"", output_name,
                             "\" conflicts with an existing intermediate result name."));
```

Reply:
done